woodwork/tests/type_system/test_ltype_inference.py

```diff
@@ -114,8 +114,21 @@ def test_categorical_inference(categories):
             assert isinstance(inferred_type, Categorical)
 
 
+def test_categorical_inference_based_on_dtype(categories_dtype):
+    """
+    This test specifically targets the case in which a series can be inferred
+    to be categorical strictly from its pandas dtype, but would otherwise be
+    inferred as some other type.
+    """
```

Comment:
I like this explanation - sometimes think it would be nice for all of our tests to have this so we are clear what they are testing months from now...